src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Sorts the projects in the project book by deadline.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_SUCCESS = "Projects are now sorted!";
```

Comment:
Maybe say here projects are now sorted by deadline will be clearer for the user. Else they might think got other criteria we sort by